dynamicProgramming/climbingStairs.py

```diff
@@ -0,0 +1,46 @@
+"""
+https://leetcode.com/submissions/detail/116364238/
+You are climbing a stair case. It takes n steps to reach to the top.
+
+Each time you can either climb 1 or 2 steps. In how many distinct ways can you climb to the top?
+"""
+class Solution(object):
+    def climbStairs(self, n):
+        """
+        :type n: int
+        :rtype: int
+        """
```

Comment:
Do you need to separate this string from the one below it? Maybe it would help to just add a `notes:` section
I know that you are following sphinx format